From: Somnath Kotur <somnath.kotur@emulex.com>
Date: Mon, 3 Oct 2011 08:10:57 +0000
Subject: [PATCH] be2net: Add 60 second delay to allow FAT dump completion on recovery from EEH
References: bnc#722092
Patch-mainline: v3.1-rc7
Git-commit: 3ab81b5f4278342a2312e3fcafa6833e40753324

Add 60s delay before timeout on polling Bit 31 so that FAT dump can
complete when reset occurs.

Signed-off-by: Somnath Kotur <somnath.kotur@emulex.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/benet/be_cmds.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/benet/be_cmds.c b/drivers/net/benet/be_cmds.c
index 4b655b8..eb58a80 100644
--- a/drivers/net/benet/be_cmds.c
+++ b/drivers/net/benet/be_cmds.c
@@ -398,7 +398,7 @@ int be_cmd_POST(struct be_adapter *adapter)
 		} else {
 			return 0;
 		}
-	} while (timeout < 40);
+	} while (timeout < 60);
 
 	dev_err(dev, "POST timeout; stage=0x%x\n", stage);
 	return -1;
-- 
1.5.6.1

